1.py

```diff
@@ -1,36 +1,28 @@
 # --- Day 1: Sonar Sweep ---
-depthlist = []
 
-while True:
```

Comment:
Remove `while True` loop since it is very dangerous to have an infinite loop anyway in your code (It can just keep running forever and consuming all the resources whilst not exiting).